hatchery/helpers.go

```diff
@@ -227,3 +229,60 @@ func stringArrayContains(s []string, e string) bool {
 	}
 	return false
 }
+
+func getLatestImageBuilderAmi(imageBuilderReaderRoleArn string, imagePipelineArn string, imagebuilderListImagePipelineImages func(*imagebuilder.ListImagePipelineImagesInput) (*imagebuilder.ListImagePipelineImagesOutput, error)) (string, error) {
+	// the `imagebuilderListImagePipelineImages` parameter should not be provided in production. It allows
+	// us to test this function by mocking `imagebuilder.ListImagePipelineImages` in the tests.
+	if imagebuilderListImagePipelineImages == nil {
+		sess := session.Must(session.NewSession(&aws.Config{
+			Region: aws.String("us-east-1"),
```

Comment:
Everywhere in hatchery, including in nextflow code, the region is hardcoded to us-east-1, i think eventually we'll want to make it configurable, it will have to be changed in many places